.github/workflows/acceptance.yml

```diff
@@ -14,11 +22,45 @@ on:
   pull_request:
 
 jobs:
+  changes:
+    runs-on: ubuntu-18.04
+    # Set job outputs to values from filter step
+    outputs:
+      backend_or_frontend: ${{ steps.filter.outputs.backend_or_frontend }}
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout sentry
+
+      - name: Check for FE or BE file changes
+        uses: getsentry/paths-filter@v2
+        id: filter
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+
+  # It is necessary to update the name and matrix in tandem with the frontend workflow
+  # to meet the required check
+  fake-frontend:
+    if: needs.changes.outputs.backend_or_frontend != 'true'
+    needs: changes
+    runs-on: ubuntu-18.04
+    name: frontend tests
+    strategy:
+      matrix:
+        instance: [0, 1, 2, 3]
+    steps:
+      - run: 'echo "Check not required." '
+
   frontend:
+    needs: changes
+    if: ${{ needs.changes.outputs.backend_or_frontend == 'true' }}
     name: frontend tests
     runs-on: ubuntu-18.04
     timeout-minutes: 20
     strategy:
+      # This allows for the workflows to continue even if one fails
+      fail-fast: false
```

Comment:
Since Github's now showing the "re-run only this check" button, I believe there's value of allowing this since FE tests are flaky these days.

You can see in this screenshot how I re-run only the failing check:
<img width="174" alt="image" src="https://user-images.githubusercontent.com/44410/159513343-b33a18d9-aee2-4c45-aaae-e6227e93059c.png">

Reply:
Over here we can see that that FE1 failing can be re-run and would cause the VS check to be triggered:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/44410/159520729-89443750-e82c-4dab-b7f0-b7f1d5801895.png">

Showing the dep graph:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/44410/159520854-af579e6e-328d-4367-bc3e-f0c105638e52.png">